ontology/dcatno.ttl

```diff
@@ -46,19 +42,21 @@
 #### ObjectProperty dcatno:service
 dcatno:service a owl:ObjectProperty ;
   rdfs:subPropertyOf dct:hasPart;
+  rdfs:label "service"@en , "tjenestebeskrivelse"@nb ;
   rdfs:domain dcat:Catalog;
   rdfs:range cpsvno:Service, cpsv:PublicService ;
-  dct:description
+  rdfs:comment
     "To include the description of a (public) service in a catalog."@en ,
     "Brukes til å inkludere beskrivelsen av en (offentlig) tjeneste i en katalog."@nb ;
   .
 
 #### ObjectProperty dcatno:event
-dcatno:service a owl:ObjectProperty ;
+dcatno:event a owl:ObjectProperty ;
   rdfs:subPropertyOf dct:hasPart;
+  rdfs:label "event"@en , "hendelsesbeskrivelse"@nb ;
```

Comment:
Kvifor er den engelske og norske label så forskjellig?
Burde det ikkje enten vere:

"event"@en , "hendelse"@nb, eller
"event description"@en, "hendeslesbeskrivelse"@nb

Reply:
Når det gjelder den engelske fulgte jeg "det etablerte mønsteret", helt fra W3C-DCAT, at det f.eks. heter bare "dataset" og ikke "dataset description" (https://www.w3.org/ns/dcat.ttl). For den norske valgte jeg å velge et mer presist navn ut fra hva egenskapene egentlig er ment for.

Reply:
Vi burde ikkje bruke labels som har ulikt meiningsinnhold på tvers av språk.

Ein annan ting eg ser no, er at dette er _egenskapen_ dcatno:service med domain dcat:Catalog. Og navnet på egenskapen er likt namnet på klassen som er range. Dette er forvirrande. Når vi først skal lage vår eigen property, kva med:
```
dcatno:containsEvent a owl:ObjectProperty;
   rdfs:label "contains event"@en, "inneholder hendelse"@nb ;
   rdfs:comment " A collection of events that is listed in the catalog."@en ;
   ...
.
```

Reply:
Ikke for å unnskylde meg, men jeg fulgte "det etablerte mønsteret" fra helt fra W3C, f.eks. dcat:Catalog dcat:dataset dcat:Dataset. 

Enig at vi burde ha valgt mer semantisk presise navn når vi først skal lage våre egne. Det gjør at jeg må oppdatere v.0.8 av CPSV-AP-NO som er ute nå.